templates/managed-namespaces-zones.tf

```diff
@@ -0,0 +1,110 @@
+{{- $allNamespaces := prepend (dict "name" "gsp-system" "ingress" (dict "enabled" true)) (datasource "config").namespaces }}
```

Comment:
IIRC when we've generated Terraform programatically before we've had a script spit out some JSON version of it. Was that considered here?